src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -37,11 +42,77 @@ def _raise_error_if_not_drawing_classifier_input_sframe(
     if len(dataset) == 0:
         raise _ToolkitError("Input Dataset is empty!")
 
-def create(input_dataset, feature="drawing", target="label", 
-            pretrained_model_url=None, classes=None, batch_size=256, 
-            num_epochs=100, max_iterations=0, verbose=True, **kwargs):
+def create(input_dataset, target, feature=None, 
+            pretrained_model_url=None, batch_size=256, 
+            num_epochs=100, max_iterations=0, verbose=True):
     """
     Create a :class:`DrawingClassifier` model.
+
+    Parameters
+    ----------
+    dataset : SFrame
+        Input data. The columns named by the ``feature`` and ``target``
+        parameters will be extracted for training the drawing classifier.
+
+    target : string
+        Name of the column containing the target variable. The values in this
+        column must be of string or integer type. String target variables are
+        automatically mapped to integers in the order in which they are 
+        provided.
+        For example, a target variable with 'square' and 'triangle' as possible
+        values is mapped to 0 and 1 respectively. Use `model.classes` to 
+        retrieve the order in which the classes are mapped.
+
+    feature : string optional
+        Name of the column containing the input drawings. 'None' (the default)
+        indicates the column in `dataset` named "drawing" should be used as the
+        feature.
+        The feature column can contain both bitmap-based drawings as well as
+        stroke-based drawings. Bitmap-based drawing input can be a grayscale
+        tc.Image of any size.
+        Stroke-based drawing input must be in the following format:
+        Every drawing must be represented by a list of strokes, where each
+        stroke must be a list of points in the order in which they were drawn
+        on the canvas.
+        Each point must be a dictionary with two keys, "x" and "y", and their
+        respective values must be numerical, i.e. either integer or float.
+
+    pretrained_model_url : string optional
+        A URL to the pretrained model that must be used for a warm start before
+        training.
+
+    batch_size: int optional
+        The number of images per training iteration. If not set, a default
```

Comment:
It seems weird to use the term "iteration" here for a single training step, whereas below iteration refers to a pass through the data. (Some of this is constrained by the precedent set by some of the pre-deep-learning toolkits, which used "iteration" for what NN practitioners probably call "epoch")

Reply:
Great catch, thank you. I'll change the wording to "training step", which is clearer and avoids confusion with `max_iterations` where iteration is supposed to mean an epoch.